sematic/ui/src/components/Notes.tsx

```diff
@@ -65,7 +65,7 @@ export function NoteView(props: {
   );
 }
 
-function RunId(props: {
+export function RunId(props: {
```

Comment:
Super nit: you might not need this change anymore. But exporting this function is not a big deal either :)

Reply:
Good catch! Reverted.